### PR TITLE
Gs/dev tiles for folder

### PR DESCRIPTION
**Pull request by GwenaelleSa**
This branch allows to import images from another folder instead of a web service.

Changes:
- modify constant names so it does not presume the input files are in the geojson format
- make a "connector" that can import images from another folder instead of a web service
- adapt the readme

This branch was not tested yet on the examples, so mistakes will have subsisted. I am doing this pull request to get the reviewers' opinion on the code, not in the idea of merging soon.

This branch is necessary to the project on the border points.

### CODE REVIEW

**Review by cleherny — COMMENTED** [review]
The new connector FOLDER looks good to me. The implementation is clear. 
I have tested it on the quarry example modulo some edits in the `config_trne.yaml` file and it works well!

File: helpers/FOLDER.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+        dirname = os.path.dirname(dst_file)
+
+        if not os.path.exists(dirname):
+            os.makedirs(dirname)
+
+        if not os.path.exists(dst_file):
+            os.link(src_file, dst_file)
+
+        return None
```
[the remark this answers is not in the document]
Comment (GwenaelleSa) [reply helpers/FOLDER.py:92]: Yes, but it can help for readability. Should I remove it?

[the remark this answers is not in the document]
Comment (cleherny) [reply helpers/FOLDER.py:87]: I don't have a strong opinion on it. According to this post: https://stackoverflow.com/questions/15300550/return-return-none-and-no-return-at-all, I would say the good practice would be to delete it but I let you decide.

File: helpers/FOLDER.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        src_file = img_file
+        dst_file = new_img_file
+
+        dirname = os.path.dirname(dst_file)
+
+        if not os.path.exists(dirname):
+            os.makedirs(dirname)
+
+        if not os.path.exists(dst_file):
+            os.link(src_file, dst_file)
```
[the remark this answers is not in the document]
Comment (cleherny) [reply helpers/FOLDER.py:85]: Ok, then let's keep it like that.

File: helpers/FOLDER.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+logger = format_logger(logger)
+
+
+
+def get_job_dict(tiles_gdf, base_path, end_path='all-images', save_metadata=False, overwrite=True):
+    """Make a dictonnary of the necessary parameters to get the tiles form a base folder and place them in the right folder.
+
+    Args:
+        tiles_gdf (GeoDataFrame): tiles with the columns x, y, and z deduced from their id
+        base_path (path): path to the original folder with the tiles
```
[the remark this answers is not in the document]
Comment (cleherny) [reply helpers/FOLDER.py:30]: I am not quite sure what I wanted to say... I guess it was replaced line 30 with "path to the original tile folder".

**Review by cleherny — APPROVED** [review]
The branch is ready to be merged (although I haven't tested it again).

**Comment by GwenaelleSa** [comment]
I tested the examples and quarry and road surface again and it is working properly.

**Pull request merged**